ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/PTP_TESTS_LEAFS.yml

```diff
@@ -66,3 +66,21 @@ l3leaf:
                   sync: 14
             priority1: 40
             priority2: 1
+
+        - name: ptp-tests-restrict-uplinks-leaf1
```

Comment:
Please move this device out of the node_group where the two others are. This is why we see so many config changes, because the other devices are no longer forming an MLAG pair...